### DISCUSSION TITLE
mspider/troubleshoot/sidecar-memory-err/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3367] [Announcements]
# mspider/troubleshoot/sidecar-memory-err/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/troubleshoot/sidecar-memory-err/

**Comment by kebe7jun** [comment]
用 Insight 的 Dashboard 的截图展示内存占用情况而不是自定义 query。
